### PR TITLE
To you, the word "QoL patch" and "Seb" are one and the same.

### PR DESCRIPTION
**Pull request by ThePainkiller**
## About The Pull Request

- Adds the Naval Surgeon formal outfit to the CBO locker

image

- New items including Naval surgeon jacket, jumpsuit, regal gloves and shoes, a bottle of special rum, a fake skull and a tricorn with a wig
- Fake skull added to artist locker for theatrical purposes
- Obscure reports now gets a randomized icon sprite on creation, picking between seven different sprites
- Swapped observer book and occult book oddities icon sprites to better fit their themes
- Marshal code handbook has a better sprite now
- New Colony Games Trophy sprite by @Dongels 
image

- Most tactical, church and hunter belts can now hold binoculars, on popular demand. Hunter belts can also hold torches now
- Fixes certain suit items (particularly suit jackets) not fitting gun cases on their suit storage slot
- Ports Eris organs sprites by MLGTASTIC#7579 (can't tag 'em for whatever reason so I'm leaving Discord handle here), with a few adaptations of my own to fit server lore
image

- Balaclavas should now properly cover the ears on their closed and open states, to avoid ears noclipping through them
- Minor typo fixes

## Changelog
:cl:
add: Added Naval Surgeon drip to CBO locker. You can get the suit and jacket from loadout too!
add: Special bottle of rum, theatrical skull (made of plastic and cruetly-free) also added to artist locker
tweak: Tactical,hunter, and church belts can now hold binoculars. Hunter belt can also hold torches now
tweak: Scabbards should now also fit on the belt slot besides being attachable as accesories
tweak: Collection of obscure reports oddity should now have a randomized sprite out of seven, previously unused sprites present in code. Marshal book has a better, more cohesive sprite now. Swapped some oddities icons to better fit what they are
fix: Balaclavas should now properly cover ears to avoid noclipping
fix: Certain storage suits (specially suit jackets) can now properly hold a gun case in their suit storage slot
imageadd: Ports organ sprites from Eris, made by MLGTASTIC, with some variations for server lore
imageadd: New Colony Games Trophy sprite by @Dongels 
imagefix: Fixes bad pixel shift on Assault Armor west facing direction
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

File: code/game/objects/items/oddities.dm [hunk 1]
```diff
@@ -153,18 +153,22 @@
...
+	icon_state = "reports1"
 	prob_perk = 45 //The skeptical mind of the inker helps keep it grounded but it shows something unable to be shaken out of the mind
 	oddity_stats = list(
 		STAT_MEC = 6,
 		STAT_COG = 6,
 		STAT_BIO = 6
 	)
 
+/obj/item/oddity/common/paper_omega/Initialize()
+	icon_state = "reports[rand(1,7)]" // All these unused sprites will now add variety to reports. - Seb
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/oddities.dm:165]: These are not unused sprites

File: code/game/objects/items/oddities.dm [hunk 2]
```diff
@@ -645,7 +649,7 @@
 /obj/item/oddity/code_book
 	name = "Marshal Code Handbook"
 	desc = "A heavy book with details and translations of the Marshal codes."
-	icon_state = "book_code"
+	icon_state = "instructional_rob" // Fits better with a supposed Marshal's logo up front. - Seb
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/oddities.dm:652]: I dislike changing it to that sprite, as the code-book one has refences to each paragraph within its sprite

File: code/game/objects/items/oddities.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/game/objects/items/oddities.dm:165]: I've looked all over code and searched far and wide, nothing that spawns naturally ever uses those icons (theta, epsilon, nuke, etc)

Comment (Trilbyspaceclone) [reply code/game/objects/items/oddities.dm:165]: sec-docs do

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
This also makes sec-docs invisable

**Comment by TheSleepyCatmom** [comment]
So why exactly are we adding something that looks like it came out of the 1400s to something over 1000 years later? The style has kinda changed since then. I'd get it if it were a costume, because people still cosplay as redcoats and revolutionaries or whatever, but why the CBO, exactly?

**Comment by Trilbyspaceclone** [comment]
The CBO like the CRO is a person of science and reason, and going back to the times when such medical was invented allows them to retrace their trainning and roots :)

**Comment by cdb-is-not-good** [comment]
> So why exactly are we adding something that looks like it came out of the 1400s to something over 1000 years later? The style has kinda changed since then. I'd get it if it were a costume, because people still cosplay as redcoats and revolutionaries or whatever, but why the CBO, exactly?

^^^^^^^^^^^^

It's fine as a costume, but positively NOT as a uniform in the CBOs locker. Yes, I get the meme - No, that's not terribly thematic or sensical. This aint /tg/.

**Comment by DorfGuy1** [comment]
There was drink called 'ships surgeon' on bay. This would go with the costume I think

**Comment by hex37** [comment]
@ThePainkiller I'd prefer the costume be in a costume kit and not the CBO locker, otherwise seems like a fine addition overall.

File: code/game/objects/items/oddities.dm (hunk 2 above)
Comment (ThePainkiller) [reply code/game/objects/items/oddities.dm:652]: image

Out of these two, the one that has the sheriff badge makes the most sense to me as a codespeak manual, that's why

**Review by hex37 — APPROVED** [review]

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 3]
```diff
@@ -188,6 +188,7 @@
 	new /obj/item/device/defib_kit/compact/combat/adv/loaded/cbo(src)
 	new /obj/item/oddity/chem_book(src)
 	new /obj/item/storage/hcases/med(src)
+	new /obj/item/storage/box/livesey(src) // Doctor Livesey. A very kind and merry person. Social personality type. Not married.
```
Comment (TheSleepyCatmom) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: Remove this. There will be no 'small change' for it. It's a costume and I don't want to place this in the head lockers.

**Review by TheSleepyCatmom — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 4]
```diff
@@ -188,6 +188,8 @@
 	new /obj/item/device/defib_kit/compact/combat/adv/loaded/cbo(src)
 	new /obj/item/oddity/chem_book(src)
 	new /obj/item/storage/hcases/med(src)
+	if(prob(25))
```
Comment (TheSleepyCatmom) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm:191]: Idk why my review used the old file but, yknow. Change this thing.

Comment (ThePainkiller) [reply code/game/objects/structures/crates_lockers/closets/secure/medical.dm:191]: Might've been since you did it before the push.

**Review by Trilbyspaceclone — APPROVED** [review]
Nothing stands out as wrong or broken anymore

File: code/game/objects/items/oddities.dm (hunk 1 above)
Comment (Trilbyspaceclone) [inline code/game/objects/items/oddities.dm:165]:
```suggestion
	icon_state = "reports[rand(1,6)]" // All these unused sprites will now add variety to reports. - Seb
```
Weres the 7th?

File: code/game/objects/items/weapons/secdocs.dm [hunk 5]
```diff
@@ -13,7 +13,7 @@
 	//var/static/inv_spawn_count = 3
 
 /obj/item/oddity/secdocs/Initialize()
-	icon_state = "paper_[pick("omega","psi","theta","nuke","delta","epsilon")]"
+	icon_state = "reports[rand(1,7)]"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/secdocs.dm:16]:
```suggestion
	icon_state = "reports[rand(1,6)]"
```

**Review by TheSleepyCatmom — APPROVED** [review]

**Review by hex37 — APPROVED** [review]

**Pull request merged**